lite/examples/sound_classification/android/app/src/main/java/org/tensorflow/lite/examples/soundclassifier/AudioBuffer.java

```diff
@@ -0,0 +1,152 @@
+package org.tensorflow.lite.examples.soundclassifier;
+
+import android.media.AudioFormat;
+import android.media.AudioRecord;
+import android.util.Log;
+
+import java.nio.FloatBuffer;
+
+public class AudioBuffer {
+
+    // TODO: What if the ring buffer is not yet fully filled before invocation?
+    private class FloatRingBuffer {
```

Comment:
Want to double confirm with the ring buffer idea. Does all audio classifier expect the ring buffer input? Are there use cases that only need a single audio clip, i.e. not a ring buffer, but a regular buffer like TensorBuffer?